src/VASL/build/module/map/ASLPieceMover.java

```diff
@@ -932,7 +932,6 @@ protected void calcDrawOffset() {}
          *
          * @param newDropWin window component to be our new draw window.
          */
-        @Deprecated(since = "2023-05-08", forRemoval = true)
         public void setDrawWinToOwnerOf(Component newDropWin) {}
```

Comment:
This doesn't fix the problem. You haven't removed the empty implementation, only the deprecation annotation.